opentripplanner-geocoder/pom.xml

```diff
@@ -121,6 +134,12 @@
       <version>2.0</version>
       <type>jar</type>
       <scope>compile</scope>
-    </dependency>
-  </dependencies>
+    </dependency>    
+    <!-- Pelias Client Library -->
+    <dependency>
+      <groupId>edu.cutr.pelias</groupId>    

```

Comment:
After my fix for the Pelias client library in https://github.com/CUTR-at-USF/pelias-client-library/pull/10, this should now be:

    <groupId>edu.usf.cutr.pelias</groupId>